ydb/core/mind/bscontroller/disk_metrics.cpp

```diff
@@ -37,6 +37,11 @@ class TBlobStorageController::TTxUpdateDiskMetrics : public TTransactionBase<TBl
             }
         }
 
+        for (auto& [vslotId, v] : Self->StaticVSlots) {
```

Comment:
it would be great to persist these metrics too (and handle VDiskMetrics table for static groups all the way along)